hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/resources/hdfs-rbf-default.xml

```diff
@@ -134,6 +134,22 @@
     </description>
   </property>
 
+  <property>
+    <name>dfs.federation.router.enable.multiple.socket</name>
+    <value>false</value>
+    <description>
+      If enable multiple downstream socket or not.
```

Comment:
We should explain the relation between dfs.federation.router.enable.multiple.socket and dfs.federation.router.max.concurrency.per.connection.

In general it would be good to have this in some of the RBF md files explaining why doing this.

Reply:
https://github.com/apache/hadoop/blob/trunk/hadoop-hdfs-project/hadoop-hdfs-rbf/src/site/markdown/HDFSRouterFederation.md